rpcs3/Emu/RSX/overlay_controls.h

```diff
@@ -457,7 +457,15 @@ namespace rsx
 					{
 						//Resource was not found in config dir, try and grab from relative path (linux)
 						info = std::make_unique<image_info>(("Icons/ui/" + res).c_str());
-
+#ifndef _WIN32
+						if (info->data == nullptr)
+						{
+							char result[ PATH_MAX ];
+							ssize_t count = readlink( "/proc/self/rpcs3", result, PATH_MAX );
+							auto executablePath = std::string( result, (count > 0) ? count : 0 );
+							info = std::make_unique<image_info>((executablePath + "../share/Icons/ui/" + res).c_str());
```

Comment:
I don't see a `dirname` call… doesn't this result in something like `/usr/local/bin/rpcs3../share/Icons/ui/something`?